vlib/v/slow_tests/inout/alias_for_c_loop_assign_op.vv

```diff
@@ -0,0 +1,10 @@
+type Float = f32
```

Comment:
Why is that in vlib/v/slow_tests/inout/ ?

It can be a simple _test.v file in `vlib/v/tests/` imho.

Reply:
Good idea